0004/README.md

```diff
@@ -0,0 +1,41 @@
+---
+id: 0004
+title: Tinkerbell Security
+status: ideation
+authors: chitrabasu khare <chitrabasukhare89@gmail.com>
+---
+
+## Summary:
+
```

Comment:
I thinik this is a NO-PROBLEM they are accessible from the outside because we made a docker-compose that can be used in a very limited amount of scenarios. Usually for testing and to try the project out.